### PR TITLE
feat(REST error codes handling): Handle 401 and 503 error on Bonita REST requests

### PR DESCRIPTION
**Pull request by abirembaut**
* replace confirm with modal (this imply using ngDialod). As far as I can tell there are no known vulnerabilities: https://security.snyk.io/package/npm/ng-dialog

Covers [DEV-545](https://bonitasoft.atlassian.net/browse/DEV-545)
Relates to [DEV-485](https://bonitasoft.atlassian.net/browse/DEV-485)
Relates to [RUNTIME-531](https://bonitasoft.atlassian.net/browse/RUNTIME-531)

[DEV-545]: https://bonitasoft.atlassian.net/browse/DEV-545?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DEV-485]: https://bonitasoft.atlassian.net/browse/DEV-485?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RUNTIME-531]: https://bonitasoft.atlassian.net/browse/RUNTIME-531?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: generator-angularjs/package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "angular-gettext": "2.4.2",
     "angular-messages": "1.3.18",
     "angular-sanitize": "bonitasoft/bower-angular-sanitize#v1.3.21",
+    "angular-ui-bootstrap": "0.14.3",
```
Comment (rbioteau) [inline generator-angularjs/package.json:74]: Can it creates conflict with existing custom widgets using angular-bootstrap ?

**Review by rbioteau — COMMENTED** [review]
Any impact on existing widgets (provided or custom) ?

File: generator-angularjs/package.json (hunk 1 above)
Comment (abirembaut) [reply generator-angularjs/package.json:74]: good point ! I will try to do it differently

File: generator-angularjs/yarn.lock [hunk 2]
```diff
@@ -481,6 +481,11 @@ angular-sanitize@bonitasoft/bower-angular-sanitize#v1.3.21:
   version "1.3.21"
   resolved "https://codeload.github.com/bonitasoft/bower-angular-sanitize/tar.gz/955a78ff88d633793d51f80a89692ac9ed8ea9c1"
 
+angular@1.3.x:
```
Comment (rbioteau) [inline generator-angularjs/yarn.lock:484]: Why this change ?
Shouldn't we use our patched version of angular ?

File: generator-angularjs/package.json (hunk 1 above)
Comment (abirembaut) [reply generator-angularjs/package.json]: I used ngDialog instead

**Comment by abirembaut** [comment]
image

**Review by Shmayro — APPROVED** [review]
lgtm
